table/metadata.go

```diff
@@ -1670,6 +1711,103 @@ func (m *metadataV2) checkLastSequenceNumber() error {
 	return nil
 }
 
+// metadataV3 represents table metadata for format version 3
+type metadataV3 struct {
+	LastSeqNum     int64 `json:"last-sequence-number"`
+	NextRowIDValue int64 `json:"next-row-id"`
+
+	commonMetadata
+}
+
+func initMetadataV3Deser() *metadataV3 {
+	return &metadataV3{
+		LastSeqNum:     -1,
+		NextRowIDValue: 0,
```

Comment:
should we also init this to -1? if we init to 0, we cannot tell if it was sent or not, 0 is a valid value, missing is not according to the spec

Reply:
Yes I think you are right about this. I have updated the PR to reflect this